content/community/index.md

```diff
@@ -13,6 +13,95 @@ A vibrant community of contributors is the reason this project exists. The [Gala
 <small>GCC2019 in Freiburg, Germany, photo by Bérénice Batut</small>
 </div>
 
+---
+
```

Comment:
Have a hook or pop-out or highlighted section here that basically redirects to 'Structure/Governance'
"Looking for how we are organised? Learn about our community structure here: "

Reply:
I think you can leave this main text here, as long as it has the button redirecting to Structure (because clicking on Community, a user might thing that will tell them the organisation of the Community, or they might want to learn more 'about' the community), so we cover both bases this way.